src/vm/corhost.cpp

```diff
@@ -476,6 +469,11 @@ HRESULT CorHost2::ExecuteAssembly(DWORD dwAppDomainId,
 
         GCPROTECT_END();
 
+        // When running under FEATURE_PAL, the SetCommandLineArgs call above will
+        // call SaveManagedCommandLine which will allocate memory using new WCHAR[]
+        // We can release this memory now.
+        //
+        ReleaseManagedCommandLine();
```

Comment:
This should not be released until ```EventPipe::Shutdown``` has been called which occurs here: https://github.com/dotnet/coreclr/blob/a9b6851ad1a9551f3f654b654b094147b50e5999/src/vm/ceemain.cpp#L1461.

Otherwise this will result in a nullptr dereference if EventPipe is left enabled at process shutdown.